unified-codes/libs/ui/src/lib/components/surfaces/organism/MenuBar.stories.tsx

```diff
@@ -6,14 +6,14 @@ import MenuItem from '../../navigation/atoms/MenuItem';
 
 export default {
   component: MenuBar,
-  title: 'MenuBar',
+  title: 'Library/MenuBar',
 };
 
-export const primary = () => {
+export const withNoProps = () => {
   return <MenuBar open={true} />;
 };
 
-export const withNoProps = () => {
+export const withControlProps = () => {
```

Comment:
Nice!

Btw, was thinking possible chance for a cheeky custom hook here!

`const { isOpen, isClosed, onOpen, onClose, onToggle } = useToggle(false);`

Can then do stuff like:

```
const { isOpen, onClose, onToggle} = useToggle(false);
return <MenuBar open={true} onClick={onToggle} onClose={onClose}
```

Not actually sure if this is appropriate here, but possibly good idea (at least, aesthetically pleasing!) to extract out simple state logic with custom hooks 😄 !

Reply:
nice! I was about to use a toggle for a new component, so that's helpful!